modules/understanding-upgrade-channels.adoc

```diff
@@ -137,7 +137,7 @@ ifndef::openshift-origin[]
 A channel can be switched from the web console or through the `adm upgrade channel` command:
 
 ----
-$ oc adm upgrade channel clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
+$ oc patch clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
```

Comment:
The double quotes behind `<channel>` is ” , which should be "
If we changed the command from "oc adm upgrade channel" to "oc patch clusterversion", line #137 should also be updated:  "through the `adm upgrade channel` command"

And I could find another two places which is using `oc adm upgrade channel ` command, should we also update them?
```
Use the web console or oc adm upgrade channel <channel> to change the update channel. You can follow the steps in Updating a cluster within a minor version by using the CLI to complete the update after you change to a 4.9 channel
```

```
If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the oc adm upgrade channel command in the OpenShift CLI to configure an empty channel
```

Reply:
cc @jianlinliu for anything I missed

Reply:
cc @jianlinliu for anything I may missed